### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by nickqweaver**

### CODE REVIEW

File: src/app/Profile/ProfileStat.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { FC } from "react";
+import Text from "../../components/ui/text/Text";
+
+type ProfileStatType = {
+  count: string;
```
Comment (sawyerb22) [inline src/app/Profile/ProfileStat.tsx]: I’d change this to stat.

File: src/components/ui/buttons/PrimaryButton.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React, { FC } from "react";
+
+type PrimaryButtonTypes = {
+  text: string;
+  onClick: () => void;
+};
+const PrimaryButton: FC<PrimaryButtonTypes> = ({ text, onClick }) => {
```
Comment (sawyerb22) [inline src/components/ui/buttons/PrimaryButton.tsx]: I’d make this component just Button. And props will control the type of button

File: src/app/Profile/ProfileStat.tsx (hunk 1 above)
Comment (nickqweaver) [reply src/app/Profile/ProfileStat.tsx]: Fixed

File: src/components/ui/buttons/PrimaryButton.tsx (hunk 2 above)
Comment (nickqweaver) [reply src/components/ui/buttons/PrimaryButton.tsx]: Done

**Pull request merged**